ui/src/adapters/api/connections.ts

```diff
@@ -70,17 +77,22 @@ export async function getConnections({
       },
       method: "GET",
       params: new URLSearchParams({
-        ...(params?.query !== undefined ? { [QUERY_SEARCH_PARAM]: params?.query } : {}),
+        ...(query !== undefined ? { [QUERY_SEARCH_PARAM]: query } : {}),
         ...(credentials ? { credentials: "true" } : {}),
+        ...(max_results !== undefined ? { max_results: max_results.toString() } : {}),
+        ...(page !== undefined ? { page: page.toString() } : {}),
       }),
       signal,
       url: `${API_VERSION}/connections`,
     });
     return buildSuccessResponse(
-      getListParser(connectionParser)
-        .transform(({ failed, successful }) => ({
-          failed,
-          successful: successful.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime()),
+      getResourceParser(connectionParser)
+        .transform(({ items: { failed, successful }, meta }) => ({
+          items: {
+            failed,
+            successful: successful.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime()),
```

Comment:
We should apply also here what Elias mentioned in [this comment](https://github.com/0xPolygonID/issuer-node/pull/564#discussion_r1431203737).

Reply:
Done

Reply:
mmm... I just realized we can indeed completely remove the `transform`. Would you please remove it also in `getCredentials`?

Reply:
sure. Removed!

Reply:
You removed the transform from `getLinks` not from `getCredentials`.

Reply:
Fixed. Transform restored in getLinks and removed from getCredentials